biztalk/install-and-config-guides/whats-new-in-biztalk-server-2020.md

```diff
@@ -14,44 +14,46 @@ author: "MandiOhlinger"
-|Additional updates|<ul><li>The [!INCLUDE[HL7_CurrentVersion_FirstRef_md](../includes/hl7-currentversion-firstref-md.md)] starts the connection to a Line of Business server (LOB), and pushes messages over the connection. The LOB waits for the connection, and then sends the messages. <br/><br/>In previous [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] versions, the HL7 MLLP receive adapter waits for the LOB server to connect to HL7, and then send messages. The LOB connects to HL7, and then sends messages. </li><br/><li>Office web components (OWC) is now optional in the installation, and listed separately in Programs</li><br/><li>The orchestration instance ID is added to the XLANG FireEvent trace output</li></ul>|   
+|Support for newer platforms|BizTalk Server 2020 adds support for the following Microsoft platforms:<br /><br /> -   Visual Studio 2019<br />-   Windows Server 2019, Windows Server 2016, Windows 10<br />-   SQL Server 2019, SQL Server 2017, SQL Server 2016 SP2<br />-   Office 2019, Office 2016<br/><br/>[Hardware and Software Requirements for BizTalk Server 2020](../install-and-config-guides/hardware-and-software-requirements-for-biztalk-server-2020.md)|  
+| Analytics | Operational Data Monitoring and Analytics. <br/>Leveraging the power of Azure (Application Insight and Event Hub) for deep data storage and Power BI for reporting and viewing of data. <br/><br/>[Track data to Azure](../core/send-tracking-data-to-azure-application-insights-using-biztalk-server.md)<br />[Configure data feed for Power BI](../core/configure-the-operational-data-feed-for-power-bi-with-biztalk-server.md)|
+| Application Lifecycle Management with VSTS | Using Visual Studio Team Services, you can define multi-server deployments of BizTalk Server 2020, and then maintain those systems throughout the application lifecycle. <br/><br/>[Configure automatic deployment using VSTS](../core/configure-automatic-deployment-with-visual-studio-team-services-in-biztalk.md)|
+| Management APIs | Manage your environment remotely using the new REST APIs with full Swagger support. <br/><br/>[Configure management REST APIs](../core/install-and-configure-the-management-rest-apis-in-biztalk-server.md)<br/>[REST API reference](https://docs.microsoft.com/en-us/rest/api/biztalk/?view=rest-biztalk-2016)|
+|Support for Always Encrypted|Use the WCF-SQL adapter to connect to SQL Server secure Always Encrypted columns. <br/><br/>[Query Always Encrypted database in SQL Server](../core/connect-to-sql-server-always-encrypted-columns-with-biztalk-server.md)|  
+|Advanced Scheduling|New and improved scheduling capabilities in Adapters.<br/><br/>[Configure time zone and recurrence](../core/configure-the-time-zone-and-recurrence-scheduling-in-biztalk-server.md)|  
```

Comment:
Some of the links may require update when we restructure the contents, specifically the ones detailed under FP.

Reply:
the files are in correct folder, restructuring will change only the left tree navigation. Hope the files will not change, otherwise we will update these to new links.